packages/config/config/devices/0x000c/hs-ms100.json

```diff
@@ -43,17 +43,21 @@
 			"valueSize": 2,
 			"unit": "seconds",
 			"minValue": 0,
-			"maxValue": 65535,
+			"maxValue": 65525,
```

Comment:
Is this really correct? Unsigned 2-byte values go from 0 to 65535

Reply:
Yes, 65525 is correct, something about the hardware.